playbooks/common/openshift-cluster/upgrades/docker/restart.yml

```diff
@@ -19,11 +19,9 @@
   when: openshift.common.is_containerized | bool
 
 - name: Wait for master API to come back online
-  become: no
-  local_action:
-    module: wait_for
-      host="{{ inventory_hostname }}"
-      state=started
-      delay=10
-      port="{{ openshift.master.api_port }}"
+  wait_for:
```

Comment:
Wait_for should be delegated, there's no guarantees that anything is accessable/resolvable outside of the cluster